roles/sriov/tasks/mlxconfig.yml

```diff
@@ -1,9 +1,19 @@
 ---
-- name: Install MFT dependencies
+- name: Install MFT dependencies (RHEL)
   ansible.builtin.yum:
-    name: "{{ sriov_os_pkgs }}"
+    name: "{{ sriov_os_pkgs_rhel }}"
```

Comment:
use the ansible.builtin.package module and then you have one task?

Reply:
Done